src/XenCert/StorageHandler.py

```diff
@@ -3413,15 +3417,19 @@ def Create(self):
 
             device_config = copy.deepcopy(self.device_config)
 
+            if 'targetIQN' in device_config and len(device_config['targetIQN'].split(',')) > 1:
+                device_config['targetIQN'] = '*'
+
             Print("      Creating the SR.")
             # try to create an SR with one of the LUNs mapped, if all fails
             # throw an exception
             for scsiId in listSCSIId:
                 try:
                     device_config['SCSIid'] = scsiId
+                    device_config_tmp = getConfigWithHiddenPassword(device_config, self.storage_conf['storage_type'])
                     XenCertPrint("The SR create parameters are {}, {}".format(
                         util.get_localhost_uuid(self.session),
-                        device_config))
+                        device_config_tmp))
```

Comment:
maybe device_config_print is more specific name for this value

Reply:
It's copy from existing code, you can search from source code.